test.js

```diff
@@ -1,26 +1,46 @@
 'use strict'
 
-var mongodb = require('mongodb')
-var MongoClient = mongodb.MongoClient
-var MongoEmitter = require('./')
-var { test } = require('tape')
-var abstractTests = require('mqemitter/abstractTest.js')
-var clean = require('mongo-clean')
-var dbname = 'mqemitter-test'
-var url = 'mongodb://127.0.0.1/' + dbname
-
-MongoClient.connect(url, { useNewUrlParser: true, useUnifiedTopology: true, w: 1 }, function (err, client) {
-  if (err) {
-    throw err
+const mongodb = require('mongodb')
+const MongoClient = mongodb.MongoClient
+const ObjectId = mongodb.ObjectId
+
+const MongoEmitter = require('./')
+const { test } = require('tape')
```

Comment:
didn't we migrate everything to the new node:test?

Reply:
I wanted to split it , just to make sure that I don't mess up the codes because I also messed up a test.
Happy to add the node:test migration to this PR, but I can also do a second PR.

Reply:
It's in now